c2corg_ui/static/js/auth/authservice.js

```diff
@@ -367,6 +383,11 @@ app.Authentication.prototype.needAuthorization = function(method, url) {
     return true;
   }
 
+  if (url.indexOf('/xreports') !== -1 && this.isAuthenticated()) {
```

Comment:
I had a problem when I wanted to show private data in the form on the EDIT page  (of my own Xreport). They were loaded only always when authorized on View page http://localhost:6558/xreports/edit/828344/en?debug
Now I fixed it little bit in the last commit.